Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uss

```diff
@@ -0,0 +1,27 @@
+﻿/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+@import "../common.uss";
+
+Image#HelpPageReportIssueIcon {
+    --unity-image: resource("Images/Icons/GitHubIcon");
+}
+
+Image#HelpPageDocumentationIcon {
+    --unity-image: resource("Images/Icons/AWSIcon");
+}
+
+Image#HelpPageVideoTutorialIcon {
+    --unity-image: resource("Images/Icons/VideoIcon");
+}
+
+Image#HelpPageForumIcon {
+    --unity-image: resource("Images/Icons/PersonIcon");
+}
+
+Image#HelpPageReportIssueExternalLink,
```

Comment:
Do these need to be different? We're not referencing them since they don't contain text and aren't clickable. Consider making them all the same name, or using the same class

Reply:
Additionally, these aren't links, but icons so make sure that's clear in the naming

Reply:
refactored